code/modules/arousal/organs/breasts.dm

```diff
@@ -73,63 +66,63 @@
 //Ridiculous sizes makes you more cumbersome.
 //this is far too lewd wah
 
-/obj/item/organ/genital/breasts/modify_size(modifier, min = -INFINITY, max = INFINITY)
-	var/new_value = clamp(cached_size + modifier, min, max)
-	if(new_value == cached_size)
+/obj/item/organ/genital/breasts/modify_size(modifier, min = -INFINITY)
+	var/new_value = clamp(size + modifier, min, max_size)
+	if(new_value == size)
```

Comment:
It's probably better to leave the max argument on the proc just in case we decide to make chems or the like that only expand to a certain size in the future